afk-daily.sh

```diff
@@ -63,7 +64,8 @@ while getopts "ce:fgi:l:s:tv:w" opt; do
         IFS=','
         for i in $OPTARG; do
             case "$i" in
-            "hoe") eventHoe=true ;;
+            "hoe") eventHoe=true ;; # Heroes of Esperia
+            "ts") eventTs=true ;;   # Treasure Scramble (same problem as HoE atm)
```

Comment:
Which "problem" are you talking about?

Reply:
It's only changing the coordinates for the arena.

Maybe one day we will "play" the events.